reactor-core/src/main/java/reactor/core/publisher/MonoMetricsFuseable.java

```diff
@@ -119,7 +105,7 @@ public void onComplete() {
 					return;
 				}
 				done = true;
-				FluxMetrics.recordOnComplete(sequenceName, commonTags, registry, subscribeToTerminateSample);
+				FluxMetrics.recordOnCompleteEmpty(sequenceName, commonTags, registry, subscribeToTerminateSample);
```

Comment:
this is not nearly enough, `poll` isn't managed and the `ASYNC` mode above isn't either